distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -31,153 +31,340 @@ deny {
+    permission java.lang.RuntimePermission "createClassLoader";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
+}
+
+grant codeBase "file:/jetty-server/catalog-core-directorymonitor/catalog-security-filter/spatial-geocoding-plugin/spatial-geocoding-geocoder/spatial-geocoding-offline-catalog/registry-identification-plugin/registry-federation-admin-service-impl" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}system.properties", "read,write";
+}
+
+grant codeBase "file:/org.apache.karaf.shell.ssh/org.apache.sshd.core" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}host.key", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}branding-ssh.properties", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}org.apache.karaf.shell.cfg", "read";
+}
+
+grant codeBase "file:/org.apache.karaf.shell.core/org.apache.shiro.core" {
+    //permission java.io.FilePermission "${ddf.home.perm}etc${/}host.key", "read";
+    //permission java.io.FilePermission "${ddf.home.perm}etc${/}org.apache.karaf.shell.cfg", "read";
+    //permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
+    //permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
+    //permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}issuer${/}-", "read";
+    //should only need the above, but the itests try to read random files, leaving for now to hopefully change in the future
+    permission java.io.FilePermission "<<ALL FILES>>", "read,write";
+}
+
+grant codeBase "file:/catalog-core-commands" {
+    permission java.io.FilePermission "<<ALL FILES>>", "read,write";
+}
+
+grant codeBase "file:/platform-osgi-configadmin" {
+    permission java.security.AllPermission;
+}
+
+grant codeBase "file:/org.apache.karaf.deployer.blueprint/org.apache.karaf.deployer.kar" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}definitions${/}-", "read";
+}
+
+grant codeBase "file:/org.apache.karaf.jaas.modules" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keys.properties", "read";
+}
+
+grant codeBase "file:/pax-logging-log4j2" {
+    permission java.io.FilePermission "${ddf.home.perm}data${/}log${/}-", "read,write";
+    permission java.io.FilePermission "${ddf.home.perm}data${/}log${/}ingest_error.log", "read,write";
```

Comment:
✏️ Should be able to remove the specific log file entries here now that `log/-` is covered.

Reply:
Going to leave for now since I had weird behavior with this bundle.